src/domain/journey/space/createSpace/CreateSpaceDialog.tsx

```diff
@@ -75,26 +81,55 @@ const CreateSpaceDialog = () => {
 
   const config = useConfig();
 
+  const [CreateNewSpace] = useCreateNewSpaceMutation();
+  const [handleSubmit] = useLoadingState(async (values: Partial<FormValues>) => {
+    if (!user?.user.id) {
+      return;
+    }
+    setDialogOpen(false);
+    setPlansTableDialogOpen(false);
+    setCreatingDialogOpen(true);
+    const newSpace = await CreateNewSpace({
+      variables: {
+        hostId: user.user.id,
+        spaceData: {
+          nameID: values.nameID,
+          profileData: {
+            displayName: values.name!, // ensured by yup validation
+            tagline: values.tagline!,
+            tagsets: values.tagsets,
+          },
+          collaborationData: {},
+        },
+      },
+    });
+
+    setCreatingDialogOpen(false);
+    //!! todo! do something with newSpace, navigate...
```

Comment:
Navigate to the space and also, let's send a warning/info log to Sentry with details about the creation: 
Category SPACE_CREATION, LABEL: planId if possible.